docs/sdk-js/overview.md

```diff
@@ -11,6 +11,12 @@
 
 Dash library for JavaScript/TypeScript ecosystem (Wallet, DAPI, Primitives, BLS, ...)
 
+:::{warning}
+The JavaScript SDK provides easy access to Dash Platform without requiring a full node;
```

Comment:
We should say that wallet in JS SDK doesn't verify data as well and on production you can use JS SDK only against trusted fullnode

Reply:
Updated in latest commit